### PR TITLE
Add Orfo 2016 16.2.2

### PR DESCRIPTION
**Pull request by nrlquaker**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/orfo.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  name 'ORFO'
+  name 'ОРФО'
+  homepage 'http://www.orfo.ru'
+
+  conflicts_with cask: 'orfo-plus'
+
+  pkg 'ORFO2016.pkg'
+
+  uninstall pkgutil:    [
+                          'com.informatic.pkg.orfo2014',
```
Comment (vitorgalvao) [inline Casks/orfo.rb]: `2014`?

File: Casks/orfo.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                          'com.informatic.pkg.seticon',
+                          'com.informatic.pkg.startpp',
+                        ],
+            delete:     '/Library/PreferencePanes/ORFOSetup.prefPane',
+            login_item: 'OrfoUpdate'
+
+  zap delete: '/Library/Application Support/ORFO 2016'
```
Comment (vitorgalvao) [inline Casks/orfo.rb]: Are you sure this is `/Library` and not `~/Library`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/orfo-2016-plus.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+cask 'orfo-plus' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.orfo.ru/download/ORFO2016Plus.dmg'
+  name 'ORFO Plus'
+  name 'ОРФО'
```
Comment (vitorgalvao) [inline Casks/orfo-2016-plus.rb]: What’s the equivalent here, to include the “Plus”?

File: Casks/orfo-2016-plus.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  uninstall pkgutil:    [
+                          'com.informatic.pkg.orfo2016',
+                          'com.informatic.pkg.seticon',
+                          'com.informatic.pkg.startpp',
+                        ],
+            delete:     '/Library/PreferencePanes/ORFOSetup.prefPane',
+            login_item: 'OrfoUpdate'
+
+  zap delete: '/Library/Application Support/ORFO 2016'
```
Comment (vitorgalvao) [inline Casks/orfo-2016-plus.rb]: Are you sure this is `/Library` and not `~/Library`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/orfo-2016-plus.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  conflicts_with cask: 'orfo'
+
+  pkg 'ORFO2016Plus.pkg'
+
+  uninstall pkgutil:    [
+                          'com.informatic.pkg.orfo2016',
+                          'com.informatic.pkg.seticon',
+                          'com.informatic.pkg.startpp',
+                        ],
+            delete:     '/Library/PreferencePanes/ORFOSetup.prefPane',
```
Comment (vitorgalvao) [inline Casks/orfo-2016-plus.rb]: Are you sure this is `/Library` and not `~/Library`?

File: Casks/orfo.rb [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  conflicts_with cask: 'orfo-plus'
+
+  pkg 'ORFO2016.pkg'
+
+  uninstall pkgutil:    [
+                          'com.informatic.pkg.orfo2014',
+                          'com.informatic.pkg.seticon',
+                          'com.informatic.pkg.startpp',
+                        ],
+            delete:     '/Library/PreferencePanes/ORFOSetup.prefPane',
```
Comment (vitorgalvao) [inline Casks/orfo.rb]: Are you sure this is `/Library` and not `~/Library`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Please don’t change the structure of the template. It’s a waste of time for you and makes it harder to read for us.

File: Casks/orfo.rb (hunk 1 above)
Comment (nrlquaker) [inline Casks/orfo.rb]: Yeap. For some reason its 2014, but 2016 in plus version.

File: Casks/orfo.rb (hunk 2 above)
Comment (nrlquaker) [inline Casks/orfo.rb]: Yes.

File: Casks/orfo-2016-plus.rb (hunk 3 above)
Comment (nrlquaker) [inline Casks/orfo-2016-plus.rb]: Плюс

File: Casks/orfo-2016-plus.rb (hunk 4 above)
Comment (nrlquaker) [inline Casks/orfo-2016-plus.rb]: Yes.

File: Casks/orfo-2016-plus.rb (hunk 5 above)
Comment (nrlquaker) [inline Casks/orfo-2016-plus.rb]: Yes.

File: Casks/orfo.rb (hunk 6 above)
Comment (nrlquaker) [inline Casks/orfo.rb]: Yes.

**Comment by nrlquaker** [comment]
Structure changed a bit (alignment) after using 'brew cask style Casks/my-new-cask.rb [--fix]'

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**